### PR TITLE
some minor tweaks

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

**Comment by harshad16** [comment]
In future, If we change KEBECHET_CONFIGURATION variable which is also present in cronJob-template  then we also have to change the varibale in kebechet/cli.py for proper functioning of the deployment.

File: README.rst [hunk 1]
```diff
@@ -72,7 +72,7 @@ To deploy kebechet on OpenShift cluster. Use the following Ansible command with
     --extra-vars=OCP_TOKEN= <openshift_cluster_token> \
     --extra-vars=KEBECHET_INFRA_NAMESPACE= <openshift_cluster_namespace> \
     --extra-vars=KEBECHET_APPLICATION_NAMESPACE= <openshift_cluster_namespace> \
-    --extra-vars=KEBECHET_CONFIGURATION= <github_repo_config.yaml> \
+    --extra-vars=KEBECHET_CONFIGURATION_PATH= <github_repo_config.yaml> \
```
Comment (fridex) [inline README.rst]: If you want to have this renamed, please change this also in sources so we are consistent - https://github.com/thoth-station/kebechet/blob/master/kebechet/cli.py#L75

Comment (harshad16) [reply README.rst]: @fridex @goern IMHO, The variable in https://github.com/thoth-station/kebechet/blob/master/kebechet/cli.py#L75 is dependent on this https://github.com/thoth-station/kebechet/blob/d1e975bc864bb55ae80dbb7511f6cbfff18f2fe6/openshift/cronJob-template.yaml#L46.

**Comment by goern** [comment]
@harshad16 please use github review -> approve for now, /-commands do not work

**Pull request merged**